templates/documentation/taspr.html

```diff
@@ -405,6 +426,89 @@ <h4>Annual mean temperature or annual total precipitation</h4>
 }
 </pre>
 
+<h4>Monthly min, mean, and maximum temperature for each year</h4>
+
+<p>Results from monthly min, mean, and maximum temperature</p>
+
+<pre>
+{
+  "historical": {
+    "CRU-TS": {
+      "historical": {
+        "April": {
+          "1901": {
+            "tasmax": "-0.5",
+            "tasmean": "-6.2",
+            "tasmin": "-12.2"
+          },
+          ...
+        },
+        ...
+      },
+      ...
+    },
+    ...
+  },
+  "projected": {
+    "5ModelAvg": {
+      "rcp45": {
+        "April": {
+          "2006": {
+            "tasmax": "1.7",
+            "tasmean": "-4.4",
+            "tasmin": "-10.6"
+          },
+          ...
+        },
+        ...
+      },
+      ...
+    },
+    ...
+  }
+}
+</pre>
+
+<p>The above output is structured like this:</p>
+
+<pre>
+{
+    historical: {
+      CRU-TS: {
+        historical: {
+          &lt;month&gt;: {
+            &lt;year&gt;: {
+              "tasmax": &lt;max tas value for the specified month during this year&gt;,
+              "tasmean": &lt;mean tas value for the specified month during this year&gt;,
+              "tasmin": &lt;min tas value for the specified month during this year&gt;;
+            },
+            ...
+          },
+          ...
+        },
+        ...
+      },
+      ...
+  },
+  projected: {
+    &lt;model&gt;: {
+      &lt;scenario&gt;: {
+        &lt;month&gt;: {
+          &lt;year&gt;: {
+            "tasmax": &lt;max tas value for the specified month during this year&gt;,
+            "tasmean": &lt;mean tas value for the specified month during this year&gt;,
+            "tasmin": &lt;min tas value for the specified month during this year&gt;;
+          },
+          ...
+        },
+        ...
+      },
+      ...
+    },
+    ...
+  }
+}
+
```

Comment:
```suggestion
</pre>
```
Just need to close the `<pre>` block here.